specs/specs.bs

```diff
@@ -125,21 +125,21 @@ There are 2 kinds of City Objects:
   - **may** have one member with the name `"geometry"`. The value is an array containing 0 or more Geometry Objects. More than one Geometry Object is used to represent several different levels-of-detail (LoDs) for the same object. However, the different Geometry Objects of a given City Object do not have to be of different LoDs.
   - **may** have one member with the name `"attributes"`. The value is an object where the attributes of the City Object are listed.
   - **may** have one member with the name `"geographicalExtent"` (the axis-aligned bounding box of the City Object). The value is an array with 6 values: `[minx, miny, minz, maxx, maxy, maxz]`
-  - **may** have one member `"children"`. The value is an array of the IDs (of type string) of the 2nd-level City Objects that are part of the 1st-level City Object. Only the direct children of the City Object are listed, not the grandchildren. Also, a City Object can have different types of City Objects as children, eg a `"Building"` can have both as children `"BuildingPart"` and `"BuildingInstallation"`. The order of the children in the array is not relevant.
+  - **may** have one member with the name `"children"`. The value is an array of the IDs (of type string) of the 2nd-level City Objects that are part of the 1st-level City Object. Only the direct children of the City Object are listed, not the grandchildren. Also, a City Object can have different types of City Objects as children, eg a `"Building"` can have both as children `"BuildingPart"` and `"BuildingInstallation"`. The order of the children in the array is not relevant.
 
 A City Object of type 2nd-level:
 
-  - **must** have one member `"parents"`. The value is an array of the IDs (of type string) of the City Objects that are its parents. For the City Objects in the CityJSON core module, this array will always be of size 1 (only one parent). New City Objects defined in extensions can have more than one parents.
-  - **may** have one member `"children"`, and the `"children"` array only references the City Objects that are one level below the current one in the hierarchy. For instance a `"BuildingPart"` that contains a `"BuildingInstallation"`.
+  - **must** have one member with the name `"parents"`. The value is an array with the IDs (of type string) of the City Object's parents. For the City Objects in the CityJSON core module, this array will always be of size 1 (only one parent). New City Objects defined in extensions can have more than one parent.
+  - **may** have one member with the name `"children"`. The value is an array with the IDs (of type string) of the City Object's children. The array references only the City Objects that are one level below the current one in the hierarchy. For instance, the `"children"` array for the object "id-1" below links to a `"BuildingPart"` but not to the `"BuildingInstallation"` which is a child of the `"BuildingPart"`.
```

Comment:
Here, I wanted to clarify that the nested children should not be  included in the array. It was unclear before with "a `"BuildingPart"` that contains a `"BuildingInstallation"`"